client/components/common/styles.module.css

```diff
@@ -0,0 +1,103 @@
+/* Dimensions */
+
+/* Full Height Container accounting for navbar */
+.fh-container {
+  min-height: calc(100vh - 4rem /* navbar height */);
+}
+
+.spacer {
+  height: var(--spacer);
+}
+
+.form-group {
+  margin-bottom: 1rem;
+}
+
+.bold-font {
+  font-weight: bold;
+}
+
+h1.modal-title {
+  font-size: var(--24px);
+  border: 0;
+  padding: 0;
+}
+
+h2.modal-subtitle {
+  font-size: var(--20px);
+  margin: 0;
+  padding: 0;
+}
+
+div.modal-inner-section-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.centered-text {
+  text-align: center;
+
+  &.vertical {
+    vertical-align: middle;
+  }
+}
+
+/* Colors */
+.dark-gray {
+  color: var(--bg-dark-gray);
+}
+
+span.required-indicator {
+  color: var(--bs-form-invalid-color);
```

Comment:
We might want to get rid of these bootstrap variables since we are maintaining our own variables now.

Reply:
Agreed! Done in https://github.com/w3c/aria-at-app/pull/1338/commits/83353dedef9dceb0a31872c847d9e11903c109d2